ratatui-core/Cargo.toml

```
@@ -22,7 +22,13 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = []
+default = ["tls-layout-cache"]
+
+## enables std support
+std = []
+
+## enables layout cache based on thread-local storage
+tls-layout-cache = ["std"]
```

I'd prefer this not use `tls` as that intuitively means Transport Layer Security to many people rather than Thread Local Storage as intended here. So perhaps:
```suggestion
thread-local-cache = ["std"]
```